2013/03/14 21:35:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Konta".' in C:\xampp\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('Konta')
#1 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company/Konta
---
2013/03/14 21:38:57 [error] [exception.CException] exception 'CException' with message 'CActiveForm and its behaviors do not have a method or closure named "select".' in C:\xampp\framework\base\CComponent.php:265
Stack trace:
#0 [internal function]: CComponent->__call('select', Array)
#1 C:\xampp\htdocs\company\protected\views\account\_form.php(20): CActiveForm->select(Object(Account), 'type')
#2 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#11 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#12 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#21 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#22 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:48:13 [error] [exception.CException] exception 'CException' with message 'Property "Account.password2" is not defined.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('password2')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('password2')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'password2')
#3 C:\xampp\framework\web\helpers\CHtml.php(1391): CHtml::activeInputField('password', Object(Account), 'password2', Array)
#4 C:\xampp\framework\web\widgets\CActiveForm.php(687): CHtml::activePasswordField(Object(Account), 'password2', Array)
#5 C:\xampp\htdocs\company\protected\views\account\_form.php(39): CActiveForm->passwordField(Object(Account), 'password2', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#15 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#16 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#26 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:49:16 [error] [exception.CException] exception 'CException' with message 'Property "Account.password2" is not defined.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('password2')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('password2')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'password2')
#3 C:\xampp\framework\web\helpers\CHtml.php(1391): CHtml::activeInputField('password', Object(Account), 'password2', Array)
#4 C:\xampp\framework\web\widgets\CActiveForm.php(687): CHtml::activePasswordField(Object(Account), 'password2', Array)
#5 C:\xampp\htdocs\company\protected\views\account\_form.php(39): CActiveForm->passwordField(Object(Account), 'password2', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#15 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#16 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#26 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:49:38 [error] [exception.CException] exception 'CException' with message 'Property "Account.password2" is not defined.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('password2')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('password2')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'password2')
#3 C:\xampp\framework\web\helpers\CHtml.php(1391): CHtml::activeInputField('password', Object(Account), 'password2', Array)
#4 C:\xampp\framework\web\widgets\CActiveForm.php(687): CHtml::activePasswordField(Object(Account), 'password2', Array)
#5 C:\xampp\htdocs\company\protected\views\account\_form.php(39): CActiveForm->passwordField(Object(Account), 'password2', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#15 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#16 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#26 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:50:06 [error] [exception.CException] exception 'CException' with message 'Property "Account.password2" is not defined.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('password2')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('password2')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'password2')
#3 C:\xampp\framework\web\helpers\CHtml.php(1391): CHtml::activeInputField('password', Object(Account), 'password2', Array)
#4 C:\xampp\framework\web\widgets\CActiveForm.php(687): CHtml::activePasswordField(Object(Account), 'password2', Array)
#5 C:\xampp\htdocs\company\protected\views\account\_form.php(39): CActiveForm->passwordField(Object(Account), 'password2', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#15 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#16 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#26 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:50:07 [error] [exception.CException] exception 'CException' with message 'Property "Account.password2" is not defined.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('password2')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('password2')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'password2')
#3 C:\xampp\framework\web\helpers\CHtml.php(1391): CHtml::activeInputField('password', Object(Account), 'password2', Array)
#4 C:\xampp\framework\web\widgets\CActiveForm.php(687): CHtml::activePasswordField(Object(Account), 'password2', Array)
#5 C:\xampp\htdocs\company\protected\views\account\_form.php(39): CActiveForm->passwordField(Object(Account), 'password2', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#15 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#16 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#26 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:50:08 [error] [exception.CException] exception 'CException' with message 'Property "Account.password2" is not defined.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('password2')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('password2')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'password2')
#3 C:\xampp\framework\web\helpers\CHtml.php(1391): CHtml::activeInputField('password', Object(Account), 'password2', Array)
#4 C:\xampp\framework\web\widgets\CActiveForm.php(687): CHtml::activePasswordField(Object(Account), 'password2', Array)
#5 C:\xampp\htdocs\company\protected\views\account\_form.php(39): CActiveForm->passwordField(Object(Account), 'password2', Array)
#6 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\company\protected\views\account\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\company\protected\controllers\AccountController.php(79): CController->render('create', Array)
#15 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#16 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/create')
#26 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/company/account/create
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 21:53:33 [warning] [application] Failed to set unsafe attribute "repeatPassword" of "Account".
in C:\xampp\htdocs\company\protected\controllers\AccountController.php (72)
in C:\xampp\htdocs\company\index.php (13)
2013/03/14 22:04:48 [warning] [application] Failed to set unsafe attribute "repeatPassword" of "Account".
in C:\xampp\htdocs\company\protected\controllers\AccountController.php (68)
in C:\xampp\htdocs\company\index.php (13)
2013/03/14 22:25:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Żądany widok "about" nie został znaleziony.' in C:\xampp\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 C:\xampp\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/company/site/page?view=about
HTTP_REFERER=http://localhost:81/company/site/index
---
2013/03/14 22:25:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Żądany widok "about" nie został znaleziony.' in C:\xampp\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 C:\xampp\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/company/site/page?view=about
HTTP_REFERER=http://localhost:81/company/site/index
---
2013/03/14 22:28:06 [error] [exception.CException] exception 'CException' with message 'Właściwość "UserIdentity.login" nie jest zdefiniowana.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\company\protected\components\UserIdentity.php(13): CComponent->__get('login')
#1 C:\xampp\htdocs\company\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\company\protected\controllers\SiteController.php(61): CModel->validate()
#6 C:\xampp\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/company/site/login
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 22:28:25 [error] [exception.CException] exception 'CException' with message 'Właściwość "Account.id" nie jest zdefiniowana.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id')
#1 C:\xampp\htdocs\company\protected\components\UserIdentity.php(19): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\company\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\company\protected\controllers\SiteController.php(61): CModel->validate()
#7 C:\xampp\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/company/site/login
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 22:28:39 [error] [exception.CException] exception 'CException' with message 'Właściwość "Account.title" nie jest zdefiniowana.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('title')
#1 C:\xampp\htdocs\company\protected\components\UserIdentity.php(20): CActiveRecord->__get('title')
#2 C:\xampp\htdocs\company\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\company\protected\controllers\SiteController.php(61): CModel->validate()
#7 C:\xampp\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/company/site/login
HTTP_REFERER=http://localhost:81/company/site/login
---
2013/03/14 22:30:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'System nie może odnaleźć żądanej akcji "account".' in C:\xampp\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\framework\web\CController.php(270): CController->missingAction('account')
#1 C:\xampp\framework\web\CWebApplication.php(282): CController->run('account')
#2 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/account')
#3 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/company/site/account
HTTP_REFERER=http://localhost:81/company/
---
2013/03/14 22:30:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'System nie może odnaleźć żądanej akcji "Konta".' in C:\xampp\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\framework\web\CController.php(270): CController->missingAction('Konta')
#1 C:\xampp\framework\web\CWebApplication.php(282): CController->run('Konta')
#2 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/Konta')
#3 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/company/site/Konta
---
2013/03/14 22:30:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'System nie może odnaleźć żądanej akcji "Konta".' in C:\xampp\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\framework\web\CController.php(270): CController->missingAction('Konta')
#1 C:\xampp\framework\web\CWebApplication.php(282): CController->run('Konta')
#2 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('site/Konta')
#3 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/company/site/Konta
---
2013/03/14 22:45:51 [error] [php] Invalid argument supplied for foreach() (C:\xampp\framework\base\CModel.php:487)
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(795): Account->update()
#1 C:\xampp\htdocs\company\protected\controllers\AccountController.php(90): Account->save()
#2 unknown(0): AccountController->actionUpdate()
#3 C:\xampp\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#4 C:\xampp\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#5 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\framework\web\filters\CFilterChain.php(133): AccountController->runAction()
#7 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\framework\web\CController.php(1145): CAccessControlFilter->filter()
#9 C:\xampp\framework\web\filters\CInlineFilter.php(58): AccountController->filterAccessControl()
#10 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#11 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\framework\web\CController.php(265): AccountController->runActionWithFilters()
#13 C:\xampp\framework\web\CWebApplication.php(282): AccountController->run()
#14 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#15 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/account/update/5
in C:\xampp\htdocs\company\protected\models\Account.php (37)
in C:\xampp\htdocs\company\protected\controllers\AccountController.php (90)
in C:\xampp\htdocs\company\index.php (13)
2013/03/14 22:52:08 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Nie jesteś upoważniony do wykonania tej akcji.' in C:\xampp\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Nie jeste?? upo...')
#1 C:\xampp\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/admin')
#9 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/company/account/admin
HTTP_REFERER=http://localhost:81/company/account
---
2013/03/14 22:53:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Nie jesteś upoważniony do wykonania tej akcji.' in C:\xampp\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Nie jeste?? upo...')
#1 C:\xampp\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account/index')
#9 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/company/account/index
HTTP_REFERER=http://localhost:81/company/account/create
---
2013/03/14 22:53:53 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Nie jesteś upoważniony do wykonania tej akcji.' in C:\xampp\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Nie jeste?? upo...')
#1 C:\xampp\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account')
#9 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/company/account
HTTP_REFERER=http://localhost:81/company/
---
2013/04/19 23:05:37 [error] [exception.CException] exception 'CException' with message 'Nieprawidłowy alias "appliaction.modules.users.models.account". Upewnij się, że wskazuje na istniejący katalog lub plik.' in C:\xampp\framework\YiiBase.php:341
Stack trace:
#0 C:\xampp\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('appliaction.mod...', true)
#1 C:\xampp\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 C:\xampp\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 C:\xampp\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 C:\xampp\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 C:\xampp\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 C:\xampp\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\framework\web\CWebApplication.php(282): CController->run('')
#11 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#12 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/company/gii/crud
HTTP_REFERER=http://localhost:81/company/gii/crud
---
2013/04/19 23:07:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Nie można ustalić żądania "account".' in C:\xampp\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('account')
#1 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company/account
HTTP_REFERER=http://localhost:81/company/account/create
---
2013/04/19 23:10:03 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Nie jesteś upoważniony do wykonania tej akcji.' in C:\xampp\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Nie jeste?? upo...')
#1 C:\xampp\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/account/a...')
#9 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/company/users/account/admin
HTTP_REFERER=http://localhost:81/company/users/account
---
2013/04/19 23:24:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Nie można ustalić żądania "users/users/account".' in C:\xampp\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/users/acc...')
#1 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company/users/users/account
HTTP_REFERER=http://localhost:81/company/users/account/create
---
2013/04/19 23:24:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Nie można ustalić żądania "users/users/account".' in C:\xampp\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/users/acc...')
#1 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company/users/users/account
HTTP_REFERER=http://localhost:81/company/users/account
---
2013/04/19 23:24:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Nie można ustalić żądania "users/users/account".' in C:\xampp\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/users/acc...')
#1 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company/users/users/account
HTTP_REFERER=http://localhost:81/company/users/account
---
2013/04/19 23:30:42 [error] [php] include(Account.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\xampp\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\company\protected\components\UserIdentity.php(13): spl_autoload_call()
#1 C:\xampp\htdocs\company\protected\models\LoginForm.php(47): UserIdentity->authenticate()
#2 C:\xampp\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#4 C:\xampp\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\company\protected\controllers\SiteController.php(61): LoginForm->validate()
#6 C:\xampp\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/site/login
in C:\xampp\htdocs\company\protected\components\UserIdentity.php (13)
in C:\xampp\htdocs\company\protected\models\LoginForm.php (47)
in C:\xampp\htdocs\company\protected\controllers\SiteController.php (61)
2013/04/19 23:34:07 [error] [php] include(Account.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\xampp\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\company\protected\components\UserIdentity.php(13): spl_autoload_call()
#1 C:\xampp\htdocs\company\protected\models\LoginForm.php(47): UserIdentity->authenticate()
#2 C:\xampp\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#4 C:\xampp\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\company\protected\controllers\SiteController.php(61): LoginForm->validate()
#6 C:\xampp\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/site/login
in C:\xampp\htdocs\company\protected\components\UserIdentity.php (13)
in C:\xampp\htdocs\company\protected\models\LoginForm.php (47)
in C:\xampp\htdocs\company\protected\controllers\SiteController.php (61)
2013/04/19 23:49:23 [error] [exception.CException] exception 'CException' with message 'Kontroler AccountController nie może znaleźć wymaganego widoku "view1".' in C:\xampp\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\framework\web\widgets\CTabView.php(214): CController->renderPartial('view1', Array)
#1 C:\xampp\framework\web\widgets\CTabView.php(146): CTabView->renderBody()
#2 C:\xampp\framework\web\CBaseController.php(173): CTabView->run()
#3 C:\xampp\htdocs\company\protected\modules\users\views\account\_form.php(91): CBaseController->widget('CTabView', Array)
#4 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\company\protected\modules\users\views\account\create.php(19): CController->renderPartial('_form', Array)
#8 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#9 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php(65): CController->render('create', Array)
#13 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#14 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#23 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/account/c...')
#24 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/company/users/account/create
HTTP_REFERER=http://localhost:81/company/users/account
---
2013/04/19 23:53:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'System nie może odnaleźć żądanej akcji "view".' in C:\xampp\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\xampp\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/account/v...')
#3 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/company/users/account/view/id/10
HTTP_REFERER=http://localhost:81/company/users/account/create
---
2013/04/19 23:57:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'System nie może odnaleźć żądanej akcji "view".' in C:\xampp\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\xampp\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/account/v...')
#3 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/company/users/account/view/id/10
HTTP_REFERER=http://localhost:81/company/users/account
---
2013/04/20 00:23:45 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\framework\web\CController.php:963)
Stack trace:
#0 C:\xampp\framework\web\CBaseController.php(130): require()
#1 C:\xampp\framework\web\CBaseController.php(95): AccountController->renderInternal()
#2 C:\xampp\framework\zii\widgets\CListView.php(281): AccountController->renderFile()
#3 C:\xampp\framework\zii\widgets\CBaseListView.php(160): CListView->renderItems()
#4 unknown(0): CListView->renderSection()
#5 C:\xampp\framework\zii\widgets\CBaseListView.php(143): preg_replace_callback()
#6 C:\xampp\framework\zii\widgets\CBaseListView.php(128): CListView->renderContent()
#7 C:\xampp\framework\web\CBaseController.php(173): CListView->run()
#8 C:\xampp\htdocs\company\protected\modules\users\views\account\index.php(21): AccountController->widget()
#9 C:\xampp\framework\web\CBaseController.php(126): require()
#10 C:\xampp\framework\web\CBaseController.php(95): AccountController->renderInternal()
#11 C:\xampp\framework\web\CController.php(869): AccountController->renderFile()
#12 C:\xampp\framework\web\CController.php(782): AccountController->renderPartial()
#13 C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php(148): AccountController->render()
#14 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionIndex()
#15 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#16 C:\xampp\framework\web\filters\CFilterChain.php(133): AccountController->runAction()
#17 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#18 C:\xampp\framework\web\CController.php(1145): CAccessControlFilter->filter()
#19 C:\xampp\framework\web\filters\CInlineFilter.php(58): AccountController->filterAccessControl()
#20 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#21 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#22 C:\xampp\framework\web\CController.php(265): AccountController->runActionWithFilters()
#23 C:\xampp\framework\web\CWebApplication.php(282): AccountController->run()
#24 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#25 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#26 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/users/account
in C:\xampp\htdocs\company\protected\modules\users\views\account\_view.php (10)
in C:\xampp\htdocs\company\protected\modules\users\views\account\index.php (21)
in C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php (148)
2013/04/20 00:32:30 [error] [exception.CException] exception 'CException' with message 'Klasa Account i jej zachowania nie posiadają metody lub domknięcia "saveModels".' in C:\xampp\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(225): CComponent->__call('saveModels', Array)
#1 [internal function]: CActiveRecord->__call('saveModels', Array)
#2 C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php(58): Account->saveModels()
#3 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionCreate()
#4 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/account/c...')
#14 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/company/users/account/create
HTTP_REFERER=http://localhost:81/company/users/account/create
---
2013/04/20 00:33:05 [error] [php] Argument 1 passed to AccountController::saveModels() must be an instance of Account, none given, called in C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php on line 57 and defined (C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php:73)
Stack trace:
#0 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\framework\web\filters\CFilterChain.php(133): AccountController->runAction()
#2 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\framework\web\CController.php(1145): CAccessControlFilter->filter()
#4 C:\xampp\framework\web\filters\CInlineFilter.php(58): AccountController->filterAccessControl()
#5 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#6 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\framework\web\CController.php(265): AccountController->runActionWithFilters()
#8 C:\xampp\framework\web\CWebApplication.php(282): AccountController->run()
#9 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/users/account/create
in C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php (73)
in C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php (57)
in C:\xampp\htdocs\company\index.php (13)
2013/04/20 01:07:57 [error] [php] get_class() expects parameter 1 to be object, array given (C:\xampp\framework\web\helpers\CHtml.php:2220)
Stack trace:
#0 C:\xampp\framework\web\helpers\CHtml.php(1255): resolveNameID()
#1 C:\xampp\framework\zii\widgets\grid\CDataColumn.php(97): activeTextField()
#2 C:\xampp\framework\zii\widgets\grid\CGridColumn.php(109): CDataColumn->renderFilterCellContent()
#3 C:\xampp\framework\zii\widgets\grid\CGridView.php(505): CDataColumn->renderFilterCell()
#4 C:\xampp\framework\zii\widgets\grid\CGridView.php(483): CGridView->renderFilter()
#5 C:\xampp\framework\zii\widgets\grid\CGridView.php(453): CGridView->renderTableHeader()
#6 C:\xampp\framework\zii\widgets\CBaseListView.php(160): CGridView->renderItems()
#7 unknown(0): CGridView->renderSection()
#8 C:\xampp\framework\zii\widgets\CBaseListView.php(143): preg_replace_callback()
#9 C:\xampp\framework\zii\widgets\CBaseListView.php(128): CGridView->renderContent()
#10 C:\xampp\framework\web\CBaseController.php(173): CGridView->run()
#11 C:\xampp\htdocs\company\protected\modules\users\views\account\admin.php(49): AccountController->widget()
#12 C:\xampp\framework\web\CBaseController.php(126): require()
#13 C:\xampp\framework\web\CBaseController.php(95): AccountController->renderInternal()
#14 C:\xampp\framework\web\CController.php(869): AccountController->renderFile()
#15 C:\xampp\framework\web\CController.php(782): AccountController->renderPartial()
#16 C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php(158): AccountController->render()
#17 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionAdmin()
#18 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#19 C:\xampp\framework\web\filters\CFilterChain.php(133): AccountController->runAction()
#20 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 C:\xampp\framework\web\CController.php(1145): CAccessControlFilter->filter()
#22 C:\xampp\framework\web\filters\CInlineFilter.php(58): AccountController->filterAccessControl()
#23 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#24 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#25 C:\xampp\framework\web\CController.php(265): AccountController->runActionWithFilters()
#26 C:\xampp\framework\web\CWebApplication.php(282): AccountController->run()
#27 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#28 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#29 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/users/account/admin
in C:\xampp\htdocs\company\protected\modules\users\views\account\admin.php (49)
in C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php (158)
in C:\xampp\htdocs\company\index.php (13)
2013/04/20 01:12:20 [error] [exception.CException] exception 'CException' with message 'Właściwość "Account.account_type.name" nie jest zdefiniowana.' in C:\xampp\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\framework\db\ar\CActiveRecord.php(143): CComponent->__get('account_type.na...')
#1 C:\xampp\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('account_type.na...')
#2 C:\xampp\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Account), 'account_type.na...')
#3 C:\xampp\framework\web\helpers\CHtml.php(1257): CHtml::activeInputField('text', Object(Account), 'account_type.na...', Array)
#4 C:\xampp\htdocs\company\protected\modules\users\views\account\admin.php(45): CHtml::activeTextField(Object(Account), 'account_type.na...')
#5 C:\xampp\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#9 C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php(158): CController->render('admin', Array)
#10 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionAdmin()
#11 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\framework\web\CWebApplication.php(282): CController->run('admin')
#20 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController('users/account/a...')
#21 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\company\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/company/users/account/admin
HTTP_REFERER=http://localhost:81/company/users/account/index
---
2013/04/20 01:15:51 [error] [php] Object of class Account could not be converted to string (C:\xampp\framework\web\helpers\CHtml.php:1918)
Stack trace:
#0 C:\xampp\htdocs\company\protected\modules\users\views\account\admin.php(45): dropDownList()
#1 C:\xampp\framework\web\CBaseController.php(126): require()
#2 C:\xampp\framework\web\CBaseController.php(95): AccountController->renderInternal()
#3 C:\xampp\framework\web\CController.php(869): AccountController->renderFile()
#4 C:\xampp\framework\web\CController.php(782): AccountController->renderPartial()
#5 C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php(158): AccountController->render()
#6 C:\xampp\framework\web\actions\CInlineAction.php(49): AccountController->actionAdmin()
#7 C:\xampp\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\framework\web\filters\CFilterChain.php(133): AccountController->runAction()
#9 C:\xampp\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\framework\web\CController.php(1145): CAccessControlFilter->filter()
#11 C:\xampp\framework\web\filters\CInlineFilter.php(58): AccountController->filterAccessControl()
#12 C:\xampp\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#13 C:\xampp\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\framework\web\CController.php(265): AccountController->runActionWithFilters()
#15 C:\xampp\framework\web\CWebApplication.php(282): AccountController->run()
#16 C:\xampp\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 C:\xampp\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\xampp\htdocs\company\index.php(13): CWebApplication->run()
REQUEST_URI=/company/users/account/admin
in C:\xampp\htdocs\company\protected\modules\users\views\account\admin.php (45)
in C:\xampp\htdocs\company\protected\modules\users\controllers\AccountController.php (158)
in C:\xampp\htdocs\company\index.php (13)
